router/router.go

```diff
@@ -67,6 +67,11 @@ func CreateRoute(h controller.Handler) *gin.Engine {
 	v1.GET("/form", verifySession(nil), h.GetAllForm)
 	v1.GET("/form/:id", verifySession(nil), h.GetFormByID)
 
+	// accesing controller by method
+	// isDoneController := controller.NewIsDoneController(db)
+	// v1.POST("/isdone/:id/status", isDoneController.ChangeStatusHandler)
+	v1.POST("/isdone/:id/status", verifySession(nil), h.ChangeStatusHandler)
```

Comment:
route should be patch or put req

/form/status is enough

Reply:
Is there a need of issue id for status confirmation?